functests-render-command/1_render_command/render_test.go

```diff
@@ -78,27 +132,48 @@ func cleanArtifacts() {
 	os.RemoveAll(assetsOutDir)
 }
 
-func runAndCompare(cmd *exec.Cmd) {
-	_, err := cmd.Output()
-	Expect(err).ToNot(HaveOccurred())
-
+func cmpRenderToRef() (string, string, error) {
```

Comment:
just a general question,  will we always have the same file structure after rendering ?  can we have for example a right outcome but in different layout on the file? (whitespaces,different lines,etc)